weiluo/Community_Contribution.Rmd

```diff
@@ -0,0 +1,125 @@
+
+#Using PostgreSQL Databse in R with MacOS Environment
+wl2824 Wei Luo
+
+### 1. Requirements
```

Comment:
please remove the numbers from all your subheadings in this file, ie: "### Requirements"